build/ndepend/sync_ndproj.py

```
@@ -0,0 +1,106 @@
+from xml.dom import Node, minidom
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--repo", type=Path, default=None)
+    args = parser.parse_args()
+    script_dir = Path(__file__).resolve().parent
+    repo = args.repo.resolve() if args.repo else script_dir.parents[1]
+    ndproj_path = repo / "braid.ndproj"
+    ndrules_path = script_dir / "braid.ndrules"
+    ndproj = minidom.parse(str(ndproj_path))
```

Ruff (0.16.2)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S318)